core/trino-main/src/test/java/io/trino/sql/planner/optimizations/TestAddExchangesPlans.java

```diff
@@ -784,6 +785,37 @@ public void testJoinNotExactlyPartitioned()
                                                 tableScan("orders"))))));
     }
 
+    // Negative test for use-exact-partitioning when colocated join is enabled (be default)
+    @Test
+    public void testJoinNotExactlyPartitioned()
+    {
+        assertDistributedPlan(
+                "SELECT\n" +
```

Comment:
We don't need to explicitly add `\n` in the sql strings.
You can also use `"""` for blocks of text in new Java version to make it more readable.